tmuxp/config.py

```diff
@@ -209,7 +209,10 @@ def expand(sconf, cwd=None, parent=None):
     if 'session_name' in sconf:
         sconf['session_name'] = expandshell(sconf['session_name'])
     if 'window_name' in sconf:
-        sconf['window_name'] = expandshell(sconf['window_name'])
+        if not (sconf['window_name'] is None):
```

Comment:
```suggestion
        if sconf['window_name'] is not None:
```

Reply:
This does not change the outcome of the test_window_title. As expected. The if statements - before and after the suggested changes - are equivalent. Were you expecting something different?

Reply:
Found something. `window_name` is being set to `tmux`. Hence the failing assertion. Now, need to dig where this is happening.